src/extension.ts

```diff
@@ -35,7 +35,7 @@ export function activate(context: ExtensionContext) {
 	let clientOptions: LanguageClientOptions = {
 		documentSelector: ['xml', 'java', 'groovy', 'kotlin', 'javascript', 'properties', 'spring-boot-properties', 'yaml'],
 		synchronize: {
-			configurationSection: ['xml', 'java', 'groovy', 'kotlin', 'javascript', 'properties', 'spring-boot-properties', 'yaml'],
+			configurationSection: ['camel', '[camel]', 'xml', 'java', 'groovy', 'kotlin', 'javascript', 'properties', 'spring-boot-properties', 'yaml'],
```

Comment:
what do the [ ] do? Can you explain?

Reply:
a bad copy-paste from xml LS client https://github.com/redhat-developer/vscode-xml/blob/6a7db6eb6eff235ef80f5fe9b0dd0ce33c9f5fc2/src/extension.ts#L76

thanks for catching it. removed

Reply:
Oh, I thought that had some special meaning. Thanks for the clarification.